### ISSUE TITLE
Movement Tests

### ISSUE THREAD

**Issue by Twonki** [Twonki/Chesskell#26]
After my Problems with the King #25 
I should really make separate Tests for every Movement of every figure. 

I´ve already done this for Pawns #7 

Just Test the Core:
- One Figure in the Middle, otherwise safekings (-> Full Movement)
- One dig blocked by friend
- One dig blocked by enemy (-> Strikeable)
- One dig blocked by end of board 

*Black and White separate*
